source/Halibut.Tests/Support/Logging/TestContextConnectionLog.cs

```diff
@@ -64,5 +66,22 @@ static LogLevel GetLogLevel(LogEvent logEvent)
                     return LogLevel.Info;
             }
         }
+
+        static LogEventLevel GetSerilogLevel(LogEvent logEvent)
```

Comment:
I feel like this should exist somewhere already, and we should just use that. But I can't find anywhere that has this 😑 Oh well 🙂